fdbclient/include/fdbclient/CommitTransaction.h

```diff
@@ -312,6 +312,8 @@ struct CommitTransactionRef {
 	bool report_conflicting_keys = false;
 	bool lock_aware = false; // set when metadata mutations are present
 	Optional<SpanContext> spanContext;
+	Optional<VectorRef<int64_t>> tenantIds; // The tenants associated with this transaction. This field only existing
```

Comment:
Since the field is used only by commit proxy, it is not mean to be used by the client. Shall we remove it from `CommitTransactionRef` and pass the tenant id separately when needed?

Reply:
This is the easiest way to implement in `addTransaction()` I think. I'm not sure it is worth to introduce more complexity in this function to save this field